Feature/json server

mimmy307 opened the pull request:
Installed json server and added db.json file

mimmy307 commented:
@ZouhairEM can you review :) ?

ZouhairEM reviewed (changes requested):
Almost perfect! However I was also expecting some content inside the db.json as well. Can you add that date which is now hardcoded into it and then in your other PR fetch it?

mimmy307 commented:
@ZouhairEM the error is now fixed. the name of the countdown file was not correct. 
I also deleted the .env file and config.js, since it's not needed in this case.
